os/fs/driver/mtd/smart.c

```diff
@@ -4408,26 +4408,6 @@ static int smart_readsector(FAR struct smart_struct_s *dev, unsigned long arg)
 		fdbg("Error reading phys sector %d\n", physsector);
 		return -EIO;
 	}
-#if SMART_STATUS_VERSION == 1
```

Comment:
What about the cases where journaling is not enabled and/or the readback wont happen while writing to flash?

Reply:
I'll add a TODO comment for the same
We develop SmartFs with journaling primarily as a robust and thorough solution that guarantees FS integrity.
Hence cases where journaling is disabled are possible to build but do not majorly concern us

Reply:
I think you can simply update it instead of TODO comment. 3~5 lines of code will be enough